tables-book/04-02-events.Rmd

```diff
@@ -328,3 +328,66 @@ body <- tabular( Heading("Patients with at least one event")*1*
 tab <- rbind(heading, body)
 useGroupLabels(tab, indent = "&emsp;", extraLines = 1)
 ```
+
+#### tidytlg
```

Comment:
This should be "### tidytlg" now -- one of the changes moved all the packages up a level.

Reply:
Thanks for pointing this out. I have fixed it in the commit below.